static-build/components/HeadMeta/index.tsx

```diff
@@ -11,29 +11,53 @@
  * limitations under the License.
  */
 import { h, Fragment } from 'preact';
-import SocialMeta from './social';
 import sharedStyles from 'css-bundle:static-build/shared/styles/index.css';
 
 import icoURL from 'asset-url:../../img/favicon.ico';
 import appleIconURL from 'asset-url:../../img/apple-touch-icon.png';
 import png32URL from 'asset-url:../../img/favicon-32x32.png';
 import png16URL from 'asset-url:../../img/favicon-16x16.png';
 import safariURL from 'asset-url:../../img/safari-pinned-tab.svg';
+import toolingReportSocialURL from 'asset-url:static-build/img/tooling-report-social.png';
+
+import config from 'consts:config';
+
+interface Props {
+  titleParts: string[];
+  description: string;
+}
+
+export default function HeadMeta({ titleParts, description }: Props) {
+  const title = [...titleParts, 'Tooling.Report'].join(' | ');
 
-export default function HeadMeta() {
   return (
     <Fragment>
       <meta
         name="viewport"
         content="width=device-width,initial-scale=1,minimum-scale=1"
       />
+      <title>{title}</title>
+      <meta property="og:title" content={title} />
```

Comment:
I merged the title into this. Didn't seem to make sense to have them seperate. You'd never have one but not the other.